docker/Dockerfile_base

```diff
@@ -35,6 +35,11 @@ ENV APP_LOG=/tmp/agent/app.log
 RUN bundle exec rake app:assets:precompile
 RUN bundle exec rails db:migrate
 
+# Just in case, clean up local.
+RUN rm ./contrast_security.yaml ./spec/dummy/contrast_security.yaml || true
+RUN rm ./*log ./spec/dummy/*log || true
+RUN rm ./agent || true
```

Comment:
Realize this is redundent vs the rm **/*log above. Can revert if desired, otherwise I'll just clean it up next time I touch it.